Sources/Build/PkgConfig.swift

```diff
@@ -103,20 +103,49 @@ struct PkgConfigParser {
         }
     }
     
-    private func parseDependencies(_ depString: String) -> [String] {
-        let exploded = depString.characters.split(separator: " ").map(String.init)
+    /// Parses `Requires: ` string into array of dependencies.
+    /// The dependecy string has seperator which can be (multiple) space or a comma.
+    /// Additionally each there can be an optional version constaint to a dependency.
+    private func parseDependencies(_ depString: String) throws -> [String] {
         let operators = ["=", "<", ">", "<=", ">="]
-        var deps = [String]()
-        var skipNext = false
-        for depString in exploded {
-            if skipNext {
-                skipNext = false
-                continue
+        let seperators = [" ", ","]
+        
+        // Look at a char at an index if present.
+        func peek(idx: Int) -> Character? {
+            guard idx <= depString.characters.count else { return nil }
```

Comment:
I think this should be an assert not a guard, isn't it a programmatic error here for idx not to be in this range?